Jenkinsfile

```diff
@@ -330,12 +304,10 @@ builders['linux-tests'] = testBuilder('linux', 'py35', '/workspace')({
         dir('dcos-cli/cli') {
             sh '''
                export PYTHONIOENCODING=utf-8; \
-               export DCOS_CONFIG=tests/data/dcos.toml; \
-               chmod 600 ${DCOS_CONFIG}; \
-               echo dcos_acs_token = \\\"${DCOS_ACS_TOKEN}\\\" >> ${DCOS_CONFIG}; \
-               cat ${DCOS_CONFIG}; \
+               dcos cluster setup ${DCOS_URL} --insecure --username=${DCOS_ADMIN_USERNAME} --password-env=DCOS_ADMIN_PASSWORD; \
```

Comment:
I see.. This was just a first WIP attempt to iterate over the CI build to fix it. After switching to the previous behavior there were an integration test failing for `dcos cluster list` (it should return one but there were none).

I haven't yet exactly figured out how to fix this, I have to investigate further.